app/instances/create_instances.py

```diff
@@ -0,0 +1,40 @@
+from app.instances.customer import Customer
+from app.instances.shop import Shop
+
+
+def transform_json_to_instances(data: dict) -> dict:
+    processed_data = {}
+
+    for key, value in data.items():
```

Comment:
No need to iterate)
You can do:
processed_data = {
    "FUEL_PRICE": data["FUEL_PRICE"],
    etc.
}